docs/technical_reference/property_models/activated_sludge_model.rst

```diff
@@ -0,0 +1,144 @@
+  "NH4 :math:`^{+}` + NH :math:`_{3}` Nitrogen, S_NH", ":math:`S_{NH}`", "S_NH", "[p]", ":math:`\text{kg/m}^{3}`"
+  "Soluble biodegradable organic nitrogen, S_ND", ":math:`S_{ND}`", "S_ND", "[p]", ":math:`\text{kg/m}^{3}`"
+  "Particulate biodegradable organic nitrogen, X_ND", ":math:`X_{ND}`", "X_ND", "[p]", ":math:`\text{kg/m}^{3}`"
+  "Alkalinity, S_ALK", ":math:`S_{ALK}`", "S_ALK", "[p]", ":math:`\text{kmol HCO}_{3}^{-}\text{/m}^{3}`"
+
+Relationships
+-------------
+.. csv-table::
+   :header: "Description", "Equation"
+
+   "Aerobic growth of heterotrophs", ":math:`ρ_1 = µ_{H}(\frac{S_{S}}{K_{S}+S_{S}})(\frac{S_{O}}{K_{O,H}+S_{O}})X_{B,H}`"
+   "Anoxic growth of heterotrophs", ":math:`ρ_2 = µ_{H}(\frac{S_{S}}{K_{S}+S_{S}})(\frac{K_{O,H}}{K_{O,H}+S_{O}})(\frac{S_{NO}}{K_{NO}+S_{NO}})η_{g}X_{B,H}`"
+   "Aerobic growth of autotrophs", ":math:`ρ_3 = µ_{A}(\frac{S_{NH}}{K_{NH}+S_{NH}})(\frac{S_{O}}{K_{O,A}+S_{O}})X_{B,A}`"
+   "Decay of heterotrophs", ":math:`ρ_4 = b_{H}X_{B,H}`"
+   "Decay of autotrophs", ":math:`ρ_5 = b_{H}X_{B,H}`"
+   "Ammonification of soluble organic nitrogen", ":math:`ρ_6 = k_{a}S_{ND}X_{B,H}`"
+   "Hydrolysis of entrapped organics", ":math:`ρ_7 = k_{H}(\frac{X_{S}/X_{B,H}}{K_{X}+(X_{S}/X_{B,H})})[(\frac{S_{O}}{K_{O,H}+S_{O}})+η_{h}(\frac{K_{O,H}}{K_{O,H}+S_{O}})(\frac{S_{NO}}{K_{NO}+S_{NO}})]X_{B,H}`"
+   "Hydrolysis of entrapped organic nitrogen", ":math:`ρ_7 = k_{H}(\frac{X_{S}/X_{B,H}}{K_{X}+(X_{S}/X_{B,H})})[(\frac{S_{O}}{K_{O,H}+S_{O}})+η_{h}(\frac{K_{O,H}}{K_{O,H}+S_{O}})(\frac{S_{NO}}{K_{NO}+S_{NO}})]X_{B,H}(X_{ND}/X_{S})`"
+
+
+Scaling
+-------
+This ASM1 property package includes support for scaling, such as providing
+default or calculating scaling factors for almost all variables. The only variables
+that do not have scaling factors are the component mass flowrate and the user will
+receive a warning if these are not set.
+
+The user can specify the scaling factors for component mass flowrates with the following:
+
+.. testsetup::
+
+  from pyomo.environ import ConcreteModel
+  from idaes.core import FlowsheetBlock
+
+.. testcode::
+
+  # relevant imports
+  import watertap.property_models.coagulation_prop_pack as props    # Needs to be replaced with ASM prop pack
+  from idaes.core.util.scaling import calculate_scaling_factors
+
+  # relevant assignments
+  m = ConcreteModel()
+  m.fs = FlowsheetBlock(default={"dynamic": False})
+  m.fs.properties = props.CoagulationParameterBlock()               # Needs to be replaced with ASM
+
+  # set scaling for component mass flowrate
+  m.fs.properties.set_default_scaling('flow_mass_phase_comp', 1, index=('Liq','H2O'))
+  m.fs.properties.set_default_scaling('flow_mass_phase_comp', 1e2, index=('Liq','TDS'))
+  m.fs.properties.set_default_scaling('flow_mass_phase_comp', 1e2, index=('Liq','TSS'))
+  m.fs.properties.set_default_scaling('flow_mass_phase_comp', 1e3, index=('Liq','Sludge'))
+
+  # calculate scaling factors
+  calculate_scaling_factors(m.fs)
```

Comment:
Maybe we can exclude this bit on scaling and maybe just include any default scaling factors.